app/services/discord.server.ts

```diff
@@ -41,22 +41,12 @@ async function fetchAllGuildMembers(
   }
 }
 
-async function getCachedGuildMembers() {
+export async function isMemberOfGuild(id: string): Promise<boolean> {
   const discordGuildMembers = await redisClient.get("discordGuildMembers");
 
-  if (discordGuildMembers) {
-    return discordGuildMembers;
-  } else {
-    const allGuildMembers = await fetchAllGuildMembers();
-    await redisClient.setEx("discordGuildMembers", 5, allGuildMembers);
-    return allGuildMembers;
-  }
-}
-
-export async function isMemberOfGuild(id: string): Promise<boolean> {
-  const discordGuildMembers = await getCachedGuildMembers();
+  const parsedResult = discordGuildMembers && JSON.parse(discordGuildMembers);
 
-  const parsedResult = JSON.parse(discordGuildMembers);
+  console.log(parsedResult, discordGuildMembers);
   const matchingUser = parsedResult.find(
```

Comment:
would `parsedResult.find` be undefined if `"discordGuildMembers"` is not in the redis cache?, if so should probs produce error 🤔  so function caller can respond with 500 etc

Reply:
Very smart, as it could be that discord goes down etc. But think this might be worth doing in a follow on PR.

Reply:
Yes or the redis cache has been reset

Reply:
Or first time app deployment